src/Database/Commands/PatchByIndexCommand.ts

```diff
@@ -29,17 +29,16 @@ export class PatchByIndexCommand extends IndexQueryBasedCommand {
   }
 
   public setResponse(response: IResponse): IRavenResponse | IRavenResponse[] | void {
-    const responseBody: IResponseBody = response.body;
-    const status: StatusCode = response.statusCode;
+    const result: IRavenResponse = <IRavenResponse>super.setResponse(response);
 
-    if(!responseBody) {
-      throw new ErrorResponseException(StringUtil.format('Could not find index {0}', this.indexName));
+    if(!response.body) {
+      throw new IndexDoesNotExistException(StringUtil.format('Could not find index {0}', this.indexName));
     }
 
-    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(status)) {
+    if (![StatusCodes.Ok, StatusCodes.Accepted].includes(response.statusCode)) {
```

Comment:
yes, i the same got it. it's a bug of vscode i think - as compiler produce no errors.
i think vscode just doesn't read all typings
anyway .includes() is found in arrays and strings: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/includes?v=control